genestack_client/scripts/genestack_user_setup.py

```diff
@@ -121,6 +125,26 @@ def input_email_and_password(host, alias=None):
     return connection, user
 
 
+def input_token(host, alias=None):
+    print 'Host: %s' % host
+    if alias:
+        msg = 'Please specify Genestack API token for "%s": ' % alias
+    else:
+        msg = 'Please specify Genestack API token: '
+    while True:
+        token = getpass(msg)
+        if not token:
+            print 'Token cannot be empty'
+            continue
+        user = User(email=None, host=host, password=None, alias=alias, token=token)
```

Comment:
I thought only token is required for authentication, without alias.

Reply:
this object is saved to config.